src/EscolaLmsQuestionnaireServiceProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace EscolaLms\Questionnaire;
+
+use EscolaLms\Core\Providers\Injectable;
+use EscolaLms\Questionnaire\Repository\Contracts\QuestionnaireRepositoryContract;
+use EscolaLms\Questionnaire\Repository\Contracts\QuestionRepositoryContract;
+use EscolaLms\Questionnaire\Repository\QuestionnaireRepository;
+use EscolaLms\Questionnaire\Repository\QuestionRepository;
+use Illuminate\Support\ServiceProvider;
+
+/**
+ * SWAGGER_VERSION
+ */
+class EscolaLmsQuestionnaireServiceProvider extends ServiceProvider
+{
+    use Injectable;
+
+    private const CONTRACTS = [
+        QuestionnaireRepositoryContract::class => QuestionnaireRepository::class,
+        QuestionRepositoryContract::class => QuestionRepository::class,
+    ];
```

Comment:
```suggestion
    public $bindings = [
        QuestionnaireRepositoryContract::class => QuestionnaireRepository::class,
        QuestionRepositoryContract::class => QuestionRepository::class,
    ];
```

https://laravel.com/docs/8.x/providers#the-bindings-and-singletons-properties

Reply:
done